### PR TITLE
[4.0] Module assignment table

### PR DESCRIPTION
**Pull request by brianteeman**
This PR corrects lots of different errors in the table for assigning modules to your menu item.

Removes ID from table cells
Adds caption to table
Adds ID to table
Adds scope to cols and rows
Moves column width style from every cell in every row to just the thead
Removes the inline style text-center from every cell in every row - text should be left aligned in columns

### Note the link to edit a module is not working and is beyond the scope of this PR see #26146

### CODE REVIEW

File: administrator/components/com_menus/tmpl/item/edit_modules.php [hunk 1]
```diff
@@ -52,22 +52,25 @@
 
 echo LayoutHelper::render('joomla.menu.edit_modules', $this); ?>
 
-<table class="table">
+<table class="table" id="modules_assigned">
+	<caption id="captionTable" class="sr-only">
+		<?php echo Text::_('COM_MENUS_MODULES_TABLE_CAPTION'); ?>
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/item/edit_modules.php:56]: Missing language string.

File: administrator/language/en-GB/en-GB.com_menus.ini [hunk 2]
```diff
@@ -144,6 +144,7 @@ COM_MENUS_MENUS_TABLE_CAPTION="Table of Menus"
 ; %1$s is for module title, %2$s is for access-title, %3$s is for position
 COM_MENUS_MODULE_ACCESS_POSITION="%1$s <small>(%2$s in %3$s)</small>"
 COM_MENUS_MODULES="Modules"
+COM_MENUS_MODULES_TABLE_CAPTION="Table of module assignements."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_menus.ini]:
```suggestion
COM_MENUS_MODULES_TABLE_CAPTION="Table of Module Assignments"
```

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_menus.ini]: My bad - thanks

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f2684ef252a970bebbcd0f32539e63345798be4b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26147">issues.joomla.org/tracker/joomla-cms/26147</a>.</sub>

**Comment by brianteeman** [comment]
conflicts resolved

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 02463464f330d1682c6d3646538c921a1a9f3476<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26147">issues.joomla.org/tracker/joomla-cms/26147</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26147">issues.joomla.org/tracker/joomla-cms/26147</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**